meetups/2024/2024-02-22-talks/program.md

```diff
@@ -0,0 +1,50 @@
+# Rust Aarhus meetup at Partisia
+
+Date: 2024-02-22
+
+Rust Aarhus invites you to this free meetup for all interested in the programming language [Rust].
+
+The program for the evening is:
+
+- 18:05 - 18:10: Welcome (5 min)
+- 18:15 - 19:00: "Building serverless apps, using WASI, WebAssembly components using Rust" by Mikkel Mørk Hegnhøj (45 min incl Q&A)
+- 19:00 - 19:10: A word from our sponsor (5-10 min)
+- 19:10 - 19:55: Light dinner sponsored by Partisia (45 min)
```

Comment:
Partisia is still figuring out who does their talk, so I think we should just add this, and we can update the agenda later with the actual speaker, if we want (for me it is also fine to just let it be as the company name).
I took 10 minutes from the lunch time, we still have 35 mins, which should be plenty time to eat a sandwich.
```suggestion
- 19:10 - 19:20: Using Rust at Partisia by Partisia (10 min)
- 19:20 - 19:55: Light dinner sponsored by Partisia (45 min)
```